docs/source/install.rst

```diff
@@ -14,7 +14,7 @@ The simplest way to install the latest release of the code is with pip. Before i
 
 .. code:: python
 
-          pip install Corrfunc
+    pip install Corrfunc
```

Comment:
Should this be `python -m pip install Corrfunc?`

Reply:
Agreed, thanks